protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -749,11 +749,39 @@ service TradingDataService {
 
 // -- Pagination --
 
-// Cursor information for cursor based pagination requests
+// All data returned from the API is ordered in a well-defined manner.
+// The specific columns and the order in which the sorting is performed
+// depend on the API endpoint being called. However, the primary sorting
+// column is usually the timestamp of the block in which the data was last updated.
+//
+// To prevent excessively large response messages and to avoid overloading
+// database resources, the API employs a cursor-based pagination mechanism.
+//
+// This Pagination message can be optionally provided as part of the request to specify:
+// - The starting point within the total result set for beginning the page
+// - The size of the returned page
+// - The ordering of the data within that page
+//
+// If no Pagination message is provided, the API will return the first page of
+// data using the default page size. The maximum page size is 1000.
```

Comment:
Is the maximum page size really 1000? Or was that supposed to say default page size is 1000?

The below gets me 2000 things in `["orders"]["edges"]`:
```
https://api.n00.devnet1.vega.xyz/api/v2/orders?pagination.first=2000
```

Reply:
IIRC we decided that the default should be 100 but the maximum 1000 - though you're right I don't see any code that enforces that!

Reply:
I see here we discussed about the page size, for graphql at least:

https://github.com/vegaprotocol/vega/issues/7291

I'll update the docs to reflect what actually happens at the moment and if/when we decide that we want a limit we can fix the docs up as well